### ISSUE TITLE
pocketsphinx_continuous

### ISSUE THREAD

**Issue by ciembor** [cmusphinx/pocketsphinx#406]
I've read this article:

https://cmusphinx.github.io/2022/08/pocketsphinx-continuous/

And I understand this decision. However I think since there is a solution, it could be still in a separate repository with a separate package. I can create my own repository and do it there, but maybe cmusphinx is a better place for it. It shouldn't be hard to maintain and it would be very useful.

**Comment by dhdaines** [comment]
Yes, seems like a good idea!  Perhaps we have something like pocketsphinx-tools or pocketsphinx-contrib...
